code-conventions.md

```diff
@@ -100,8 +100,17 @@ If we need to remove or alter contributed code due to a licensing issue we will
   - When omitting brackets, still indent as if the code had brackets
   - Avoid omitting brackets if it produces hard-to-read code
 * Annotations for methods and classes are placed in lines before their declarations, one per line
+  - Annotations affecting the return type should be placed before the return type if it doesn't hurt readability.
+    ```java
+    @Nullable String motd();
+    ```
+    Is good, whereas:
```

Comment:
Can this be like this instead

Good Example:
```java
    @Nullable String motd();
```
Bad Example:
```java
    @Contract("null -> null; !null -> !null") Skript instance(Plugin plugin);
```
in the above the contract should be placed on the line before.